src/analysis/retail/shaman/elemental/modules/talents/MasterOfTheElements.tsx

```diff
@@ -14,27 +15,27 @@ const MASTER_OF_THE_ELEMENTS = {
   DURATION: 15000,
   WINDOW_DURATION: 500,
   AFFECTED_DAMAGE: [
-    SPELLS.ICEFURY_TALENT,
+    TALENTS.ICEFURY_TALENT,
     SPELLS.ICEFURY_OVERLOAD,
-    SPELLS.FROST_SHOCK,
+    TALENTS.LAVA_BURST_TALENT,
```

Comment:
also frost shock -> lava burst error here

Reply:
changed to FROST_SHOCK